### PR TITLE
Add GitHub repos plugin to plugin directory

### PR DESCRIPTION
**Pull request by gabrielpoca**

### CODE REVIEW

**Comment by gabrielpoca** [comment]
Cool project! I just found out about it yesterday, and I decided to contribute with one of my favorite workflows. The link is [here](https://github.com/gabrielpoca/zazu-github-repos). Please let me know if there is something I should change.

**Comment by bayleedev** [comment]
Great contribution, let me try it out :D

File: docs/_packages/github-repos.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+top:         true
+type:        plugin
+title:       "GitHub repos"
+image:       /images/package-icons/github-repos.png
+githuburl:   gabrielpoca/zazu-github-repos
+description: "Zazu plugin to search and open your GitHub repos"
```
Comment (bayleedev) [inline docs/_packages/github-repos.md]: Can you remove "zazu" from the sentence? Maybe "Search and open your GitHub repos".

**Comment by bayleedev** [comment]
screen shot 2017-01-29 at 10 17 04 am

Shiny!

File: docs/_packages/github-repos.md (hunk 1 above)
Comment (gabrielpoca) [reply docs/_packages/github-repos.md]: Yes :+1: updated :)

**Comment by bayleedev** [comment]
plugins

it's live [:

**Comment by amjith** [comment]
If I want this to work with Github enterprise do I just replace the `machine` entry in `~/.netrc` with the appropriate GHE endpoint?

**Comment by gabrielpoca** [comment]
Hi @amjith , I've to make some changes; I'll post an update soon  👍

**Pull request merged**